### ISSUE TITLE
[bug]: Update to Docker command breaks production installs

### ISSUE THREAD

**Issue by Fmstrat** [makeplane/plane#4646]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

Recently the `stable` docker images changed their command structure. For instance, `command: /usr/local/bin/start.sh web/server.js web` was changed to `command: node web/server.js web` in the compose files.

This is a breaking change for any user hosting via docker. `command` shouldn't be an option used in `compose` unless you are trying to alter the intended function. `stable` images should use `CMD` in the `Dockerfile`, and `command` should be removed from the compose files to ensure future iterations of images use the appropriate startup.

If the `plane-backend` wishes to be reused for low-size, a base `plane-backend` image should be built, and a `plane-api` image should be built `FROM plane-backend` to allow for separate `CMD` use.

### Steps to reproduce

1. Docker pull the latest `stable` from an install prior to this commit: https://github.com/makeplane/plane/commit/1c377163d2b184fae5806ee33ef0325d79e51ade
2. Restart
3. Watch production go down

### Environment

Production

### Browser

None

### Variant

Self-hosted

### Version

`stable`

**Comment by mguptahub** [comment]
@Fmstrat 

I agree with what you said about the CMD must be at docker image level. We have noted it and will fix it in upcoming releases.

**Comment by akshat5302** [comment]
Fixed in v0.28.0 by #7476: the frontend images (web, space, admin, live) now start from their own `CMD`, and the compose file no longer overrides their commands. Backend services still select their role with `command:` because they share the `plane-backend` image. Closing as completed. If you still hit a startup break on a current release, please open a new issue with the details.
